StreamVideoTests/Mock/CallParticipant_Mock.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+import WebRTC
+
+extension CallParticipant {
+
+    static func dummy(
```

Comment:
It seems we have `makeCallParticipant` in the `MockResponseBuilder` that does pretty much the same thing. We should maybe align them, doesn't have to be in this PR.

Reply:
Done!

Reply:
Done!